template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/CMRepositoryVersionUtil.java

```diff
@@ -74,6 +83,22 @@ public class CMRepositoryVersionUtil {
 
     public static final Versioned CDPD_VERSION_7_2_11 = () -> "7.2.11";
 
+    public static final Map<CloudPlatform, Versioned> MIN_CM_VERSION_FOR_RAZ = new HashMap<>() {
+        {
+            put(AWS, CMRepositoryVersionUtil.CLOUDERAMANAGER_VERSION_7_2_2);
+            put(AZURE, CMRepositoryVersionUtil.CLOUDERAMANAGER_VERSION_7_2_2);
```

Comment:
again, wrong versions

Reply:
it has been updated as part of this change